apps/src/studio/studio.js

```diff
@@ -3315,8 +3333,8 @@ Studio.setBackground = function (opts) {
 
     // Draw the tiles (again) now that we know which background we're using.
     if (level.wallMapCollisions) {
-      $(".tile_clip").remove();
-      $(".tile").remove();
+      $(".specificTileClip").remove();
```

Comment:
This was my first attempt to separate the tile sets, and I left it in in case it would be helpful in a future where both level-supplied and skin-supplied wall maps would coexist.  But you're right that it's now superfluous, so I can revert it to keep things simpler.